canonical-web-and-design/design.ubuntu.com
# Remove pictograms section and any references to it

## Done

- Deletes the 'pictograms' index.md
- Updates text to not reference pictograms

## QA

- Check that the page doesn't exist and there are no link attempting to lead there.
- Check there are no references to pictograms anywhere.
- Check the text changes still make sense and read well.

## Issue / Card

https://github.com/canonical-web-and-design/design.ubuntu.com/issues/211

## Screenshots

[if relevant, include a screenshot]
